larq_compute_engine/core/bconv2d_impl_ref.h

```diff
@@ -0,0 +1,126 @@
+#ifndef COMPUTE_EGNINE_CORE_BCONV2d_IMPL_H_
```

Comment:
I know there is a comment just below saying where this file comes from but if it's derived from the original in TF I think we need to keep the copyright notice, as we do elsewhere in LCE.

```suggestion
/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
Modifications copyright (C) 2020 Larq Contributors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
==============================================================================*/

#ifndef COMPUTE_EGNINE_CORE_BCONV2d_IMPL_H_
```